_templates/faqs.md

```diff
@@ -9,14 +9,26 @@ tags: [Relevant keywords. A list of the available KB tags is available in the ta
 teampulseid: [A number, representing the TeamPulse ID of the issue\question. Leave this blank if the issue\question doesn't exist as an item in TeamPulse]
 ticketid: [A number, representing the ticket ID associated with this issue\question. The field can contain more than one ticket IDs (e.g. several clients have reported it). In such cases, we separate the ticket IDs with commas. Leave this blank if the issue\question is not related to a support case]
 pitsid: [A number, representing the public issue tracker ID associated with the issue\question. Leave this blank for now as PITS is not developed yet]
-
 ---
 
 ## Environment
 <table>
   <tr>
     <td>Service</td>
-    <td>Code (AppBuilder), Views, App Distribution (AppManager), Data (Backend Services), Users, Notifications, Business Logic, Analytics, Feedback, Samples, Telerik Platform General, Third-party product</td>
+    <td>
+{{site.ab-sab}}, <!--Code (AppBuilder)-->
```

Comment:
If you agree with my comments about the _config.yml, make this in two parts (like the BS one).